quantecon/markov/random.py

```diff
@@ -218,7 +219,8 @@ def random_discrete_dp(num_states, num_actions, beta=None,
     ddp = DiscreteDP(R, Q, beta, s_indices, a_indices)
     return ddp
 
-
+#TODO: Should this use (nopython=True) flag?
```

Comment:
`nopython=True` should work if `dtype=int` is replaced with `dtype=np.int64`. (Again I suspect we won't have much speed up.)